python-primitiv/tests/model.py

```diff
@@ -144,3 +134,31 @@ def test_model_get_all_parameters(self):
         self.assertIs(params[("p2",)], parentmodel.p2)
         self.assertIs(params[("sub", "sp1")], parentmodel.sub.sp1)
```

Comment:
Redundant parenthesis?

Reply:
Of course it works without parentheses if tuples contain two or more items.
However `params` is just a dictionary, so it does not work with a single string.

For consistency, it is better to write parentheses.

Reply:
I meant that this comment points only lines that use 2-tuples, but your argument is also appropriate.